daemon/analytics.c

```diff
@@ -335,6 +357,84 @@ void analytics_alarms_notifications(void)
     buffer_free(b);
 }
 
+/*
+ * Checks for the existance of .install_type file and reads it
+ */
+void analytics_get_install_type(void)
+{
+    char *install_type_filename;
+    analytics_set_data_str(&analytics_data.netdata_install_type, "");
+    analytics_set_data_str(&analytics_data.netdata_prebuilt_distro, "");
+
+    install_type_filename =
+        mallocz(sizeof(char) * (strlen(netdata_configured_user_config_dir) + strlen(".install-type") + 2));
+    sprintf(install_type_filename, "%s/%s", netdata_configured_user_config_dir, ".install-type");
+    if (unlikely(access(install_type_filename, R_OK) != 0)) {
+        freez(install_type_filename);
+        return;
+    }
+
+    FILE *fp = fopen(install_type_filename, "r");
+    if (fp) {
+        char *s, *t, buf[256 + 1];
+        size_t len = 0;
+
+        while ((s = fgets_trim_len(buf, 4096, fp, &len))) {
+            if (!strncmp(buf, "INSTALL_TYPE='", 14)) {
+                s = t = buf + 13;
+                if (s) {
+                    while (*s == '\'')
+                        s++;
+                    while (*++t != '\0')
+                        ;
```

Comment:
Please, move this semicolon for the previous line.

Reply:
Thanks! changed with https://github.com/netdata/netdata/pull/11237/commits/8d59d9032b746dbaf1b643522303bfcf79b52a06